docs/docs/concepts/polling-profiles.mdx

```diff
@@ -0,0 +1,51 @@
+---
+id: polling-profiles
+title: Polling Profiles
+description: Polling profiles define how Tracetest will interact with a Tracing Backend to retrieve traces from tests
+keywords:
+  - tracetest
+  - trace-based testing
+  - observability
+  - distributed tracing
+  - testing
+image: https://res.cloudinary.com/djwdcmwdz/image/upload/v1698686403/docs/Blog_Thumbnail_14_rsvkmo.jpg
+---
+
+When running a trace-based testing, a crucial thing to have is the trace generated by the test execution. Usually, these traces are stored
+in a tracing backend (Jaeger, Tempo, etc) or they can be sent directly to Tracetest via OpenTelemetry Collector. In both cases, the trace is not
+complete as soon as the test is run, the application might run a long-running operation or it's batching generated spans before sending them
+to the tracing backend. In any case, spans are not sent instantaneous and are not available for Tracetest as soon as the test trigger is
+done running.
+
+This means that Tracetest needs to check periodically if the trace is complete or not. For this, it uses a `Polling Profile`. This profile
+tells Tracetest how often and how long to wait for a trace to be done. Every Tracetest environment has one default polling profile, it means that any
+test will use that profile when trying to fetch the trace from the tracing backend. However, since version v1.0.0, users can now specify a different
```

Comment:
```suggestion
test will use that profile when trying to fetch the trace from the tracing backend. Since version v1.0.0, users can now specify a different
```